calicoctl/commands/node/run.go

```diff
@@ -192,12 +192,17 @@ Description:
 
 	// Create a mapping of environment variables to values.
 	envs := map[string]string{
-		"NODENAME": name,
+		"NODENAME":                  name,
 		"CALICO_NETWORKING_BACKEND": backend,
 		"NO_DEFAULT_POOLS":          noPoolsString,
 		"CALICO_LIBNETWORK_ENABLED": fmt.Sprint(!disableDockerNw),
 	}
 
+	// Set CALICO_LIBNETWORK_IFPREFIX env variable to "cali" by default if Docker network is enabled.
+	if !disableDockerNw {
+		envs["CALICO_LIBNETWORK_IFPREFIX"] = "cali"
```

Comment:
Not sure I understand what you mean by "configurable" here, do you mean just leave it empty?